api/libproject/generate_appengine_web_xml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+CIRCLECI="${CIRCLECI:-false}"
+
+if [[ ! "$CIRCLECI" == "true" ]]; then
+  cd db
+  source vars.env
```

Comment:
How does vars.env wind up in Circle?

We want to use environment variables set up in the CI console...